databricks-sdk-java/src/main/java/com/databricks/sdk/core/GrpcTranscodingQueryParamsSerializer.java

```diff
@@ -103,24 +97,30 @@ private static String getFieldName(Field f) {
     }
   }
 
-  private static Map<String, Object> flattenObject(Object o) {
+  private static Map<String, Object> flattenObject(Object o, Boolean onlyAnnotatedFields) {
```

Comment:
Since the test is dependent on another PR, should we add a unit tests for this?

Reply:
I could have... but the other PR should be merged in the next day or two, so I think we can deal with that.